makes/container-image/main.nix

```diff
@@ -30,13 +30,28 @@ __nixpkgs__.dockerTools.buildImage {
       __nixpkgs__.gnutar
       __nixpkgs__.gzip
       __nixpkgs__.nixVersions.nix_2_15
+      __nixpkgs__.util-linux
```

Comment:
I think this is unnecessary. You can directly run commands via `runCommand` in order to configure swap.